devtools/conda-envs/beta_rc_env.yaml

```diff
@@ -0,0 +1,47 @@
+name: test
+channels:
+  - conda-forge/label/openmm_rc
+  - openeye/label/beta
+  - openeye/label/rc
```

Comment:
I'm not confident in this; if I have them in the other order, an old RC will be picked up instead of newer betas. This order ensures betas are picked up when they're newer, but I'm not sure what happens when an RC is newer than a beta.

Reply:
(non blocking) I saw you CC'ed me on an email to them. Let's stick with this ordering -- Maybe their response will let us reduce down to just one channel/label.